docs/gethue/content/en/posts/2021-08-10-open-in-importer-and-copy-path-options-in-filebrowser.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Open In Importer and Copy Path Options in Filebrowser
+author: Hue Team
+type: post
+date: 2021-08-10T00:00:00+00:00
+url: /blog/2021-08-10-open-in-importer-and-copy-path-options-in-filebrowser
+sf_thumbnail_type:
+  - none
+sf_thumbnail_link_type:
+  - link_to_post
+sf_detail_type:
+  - none
+sf_page_title:
+  - 1
+sf_page_title_style:
+  - standard
+sf_no_breadcrumbs:
+  - 1
+sf_page_title_bg:
+  - none
+sf_page_title_text_style:
+  - light
+sf_background_image_size:
+  - cover
+sf_social_sharing:
+  - 1
+sf_related_articles:
+  - 1
+sf_sidebar_config:
+  - left-sidebar
+sf_left_sidebar:
+  - Sidebar-2
+sf_right_sidebar:
+  - Sidebar-1
+sf_caption_position:
+  - caption-right
+sf_remove_promo_bar:
+  - 1
+ampforwp-amp-on-off:
+  - default
+categories:
+  - Version 4.11
+  - Development
+  - Query
+
+---
+
+As we are working to provide the most pleasant user interface to our customers so moving in this direction we are adding some more features in filebrowser. Now you can create a table from filebrowser, just select a file and click the "Open in Importer" option and also you can copy the path of a file by clicking the "Copy Path" option in fileborwser.
```

Comment:
Maybe skip first sentence and instead say we simplify object storage and DW integration?

Reply:
done